### DISCUSSION TITLE
[Android] How to automatically have Rustdesk service start on boot?

### DISCUSSION THREAD

**Discussion by stephaniehe8080** [rustdesk/rustdesk#8973] [Q&A] [answered]
I have enabled to autoboot setting in the Rustdesk apk but when I power off the Android android and boot up, I need to enter my password once to unlock before the Rustdesk service starts

**Comment by stephaniehe8080** [comment]
Used adb commands to simulate the unlock of Android

**Comment by rustdesk** [comment, marked as answer]
https://github.com/rustdesk/rustdesk/wiki/FAQ#android-start-on-boot-and-limitations

any further discussion please go to https://github.com/rustdesk/rustdesk/discussions/1330
